jobs/sartregroup_NY_pythondatadev.html

```diff
@@ -0,0 +1,58 @@
+---
+title: Python Developer - HFT
+company: Sartre Group (Agency) on behalf of GTS.
+location: New York, US
+contract: Permanent
+contact:
+    name: Joe Jolly
+    email: joe.jolly@sartregroup.com
+    phone: +1 917 795 0584
+created: !!timestamp '2017-01-25'
+tags:
+  - python
+  - expert
+  - scientific libraries
+  - numpy
+  - pandas
+  - scipy
+  - numpy
+  - trading analytics
+  - HFT
+  - trade analysis
+  - data analysis
+  - quant finance
+---
+
+# **Overview**
+
+Python developer role, working in a leading financial institution (see company description).
+
+The role is placed within a tight-knit engineering team, working on cutting edge data processing systems of trading activities.
+
+The position involves expert level python development, as you take ownership of a high volume data-processing solution. This system will perform compliance verification on a daily basis of the entire firms trading activities across all asset classes. This system will process billions of records per day, calculating P&L amongst other metrics. Work will also include analysing data (historical and financial) from various sources. There will also be high levels of collaboration with trading model developers and management. 
```

Comment:
Can you escape the & symbols here - remember this gets compiled to HTML. One day we will automate this, for for now you need to write it as "&amp;"